tests/noviswitch.py

```diff
@@ -0,0 +1,604 @@
+import os
+import re
+import json
+import paramiko
+import time
+from mininet.node import Switch
+from mininet.clean import addCleanupCallback
+import logging
+import logging.handlers
+
+
+novi_cleanup_commands = [
+    "del config controller controllergroup all controllerid all",
+    "del config flow tableid all",
+]
+
+formatter = logging.Formatter(fmt="%(asctime)s %(name)s - %(levelname)s - %(message)s")
+handler = logging.handlers.SysLogHandler("/dev/log")
+handler.setFormatter(formatter)
+logger = logging.getLogger("noviswitch")
+logger.setLevel(logging.DEBUG)
+logger.addHandler(handler)
+
+
+def run_cmd(cmd):
+    logger.debug("running cmd in %s: %s" % (NOVILOCALIP, cmd))
+    stream = os.popen(cmd)
+    return stream.read()
+
+
+NOVISETTINGS = os.environ.get("NOVISETTINGS", {})
```

Comment:
It'd be nice to have `NOVISETTINGS` and `NOVISWITCHES` expected value/format in the markdown docs too, wdyt? Also mentioning that `SWITCH_CLASS` needs to be set as "NoviSwitch" when using the NoviFlow virtual switch backend

Reply:
Awesome, thanks, Italo